### PR TITLE
Multiple features and bug eliminations

### PR DESCRIPTION
**Pull request by brunovcosta**
I just added those features/bug eliminations
- Select tag
- Tab navigation #3 
- Checkbox only possible #6 
- No interference with commom sweetAlerts #5

### CODE REVIEW

**Comment by taromero** [comment]
Thanks for all the work. Please check compliance with feross/standard style. After that I'll test the changes.

**Comment by brunovcosta** [comment]
feross/standard style checked

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,226 @@
+0 info it worked if it ends with ok
```
Comment (taromero) [inline npm-debug.log]: Please remove this file from the last commit. This should not be under version control. I would revert the last commit with `git reset HEAD~1` and then add npm-debug.log to .gitignore. Then commit again and push force.

File: live-demo/live-demo.js [hunk 2]
```diff
@@ -37,12 +37,20 @@ function complex () {
...
+            {value:"test1",text: "test1"},
+            {value:"test2",text: "test2"},
+            {value:"test3",text: "test3"},
+            {value:"test4",text: "test4"},
+            {value:"test5",text: "test5"}
+          ]},
+      ]
+  }, function(isConfirm) {
+      // do whatever you want with the form data
+      console.log(this.swalForm); // { name: 'user name', nickname: 'what the user sends' }
```
Comment (taromero) [inline live-demo/live-demo.js]: Please review feross/standard compliance. This particular line for example is not compliant (as of the semicolon). To check compliance you need to first install feross/standard using `npm install standard` and then run the install command (everything within project's directory)/

Comment (brunovcosta) [inline live-demo/live-demo.js]: removed npm-debug from version control

File: npm-debug.log (hunk 1 above)
Comment (brunovcosta) [inline npm-debug.log]: removed npm-debug.log from version control

File: live-demo/live-demo.js (hunk 2 above)
Comment (brunovcosta) [inline live-demo/live-demo.js]: done

**Comment by taromero** [comment]
Merged, thanks for contributing!

**Pull request merged**